### PR TITLE
[PREVIEW]Feature/sscs 3653

### PR DESCRIPTION
**Pull request by iguannago**

### CODE REVIEW

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/config/NotificationConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    @Rule
+    public final SpringMethodRule springMethodRule = new SpringMethodRule();
+
+    @Autowired
+    private NotificationConfig notificationConfig;
+
+    @Test
+    @Parameters({"a64bce9a-9162-47ca-b3e7-cf5f85ca7bdc, f5b61f94-0b2b-4e8e-9c25-56e9830df7d4, PIP",
+            "a64bce9a-9162-47ca-b3e7-cf5f85ca7bdc, f5b61f94-0b2b-4e8e-9c25-56e9830df7d4, ESA"})
+    public void should_getDwpResponseTemplate_when_aDwpResponseNotificationPaperCaseHappens(
```
Comment (jackmaloney) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/config/NotificationConfigTest.java]: There is already an integration test for this scenario in `NotificationsIT` called `shouldNotSendNotificationForAnResponseReceivedRequestForAPaperHearing` so believe we can remove this. Obviously you would need to fix the test as part of this scenario

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -151,6 +150,10 @@ notification:
     responseReceived:
       emailId: 90f0ed29-a616-4ce0-b4ef-108391f5d90e
       smsId: e2e166c4-3600-443d-8feb-39f2c28e8732
+  paper:
```
Comment (jackmaloney) [inline src/main/resources/application.yaml]: Could we be clear that paper and oral cases are different notification types? So add `ORAL` to `AppealHearingType.java` and then structure the 2 different types of notifications in this yaml file so we can clearly differentiate between `paper.responseReceived.emailId` and `oral.responseReceived.emailId`

Comment (iguannago) [reply src/main/resources/application.yaml]: done

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java [hunk 3]
```diff
@@ -417,8 +442,10 @@ public void shouldSendSubscriptionUpdatedNotificationForSubscriptionUpdatedReque
     @Test
     public void shouldSendSubscriptionUpdatedNotificationForSubscriptionUpdatedRequestWithNewEmailAddressForAPaperHearing() throws Exception {
         updateJsonForPaperHearing();
-        json = json.replace("appealReceived", "subscriptionUpdated");
-        json = updateEmbeddedJson(json, "No", "case_details", "case_data", "subscriptions", "appellantSubscription", "subscribeSms");
+        json = updateEmbeddedJson(json, "subscriptionUpdated", "event_id");
+        json = updateEmbeddedJson(json, "paper", "case_details", "case_data", "appeal", "hearingType");
```
Comment (jackmaloney) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java]: Looks like maybe you are doing the same thing twice now. `updateJsonForPaperHearing()` also updates this line in json

**Pull request merged**